cypress/e2e/home-page.cy.js

```diff
@@ -330,7 +330,7 @@ describe('Home page', () => {
 
     context('When they have work orders attached to them', () => {
       beforeEach(() => {
-        cy.clock(new Date('June 11 2021 13:49:15Z'))
+        // cy.clock(new Date('June 11 2021 13:49:15Z'))
```

Comment:
Flagging as the description only mentions `cy.audit`.

Reply:
forgot to remove